react/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.1'
+
+services:
+  backstop:
+    image: backstopjs/backstopjs:v3.1.21
+    volumes:
+      - ./:/src
+    depends_on:
+      - web
+    shm_size: 1gb
+    entrypoint: [backstop]
```

Comment:
Took me a sec to spot what you're doing with the configs (two different ones).  Totally up to you, but I find it easier to allow some variance within an individual config by allowing flags.  Eg: https://github.com/massgov/mass/blob/develop/backstop/backstop.js#L4-L6

Using something like that you can use the same config with flags (`--atoms` could trigger the addition of the atom patterns, etc).  Less code to manage, and cleaner commands.

Reply:
Oh, I think I was missing the point here -- yes, I think I will try that ^.^ thanks again